xdp-onebranch.sln

```diff
@@ -0,0 +1,235 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.8.34408.163
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "nmr", "src\nmr\nmr.vcxproj", "{F389859E-1D0C-4F30-BAEC-6A5E44B8190E}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "pollshim", "src\pollshim\pollshim.vcxproj", "{4A54653B-B380-4187-890B-7451F835B07A}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "rtl", "src\rtl\rtl.vcxproj", "{043C2162-639F-4FC8-B72C-F7C1BACB9DB3}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "xdplwf", "src\xdplwf\xdplwf.vcxproj", "{63591DCB-0BBC-441C-A8A2-F882EE6A7D5C}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "xdpapi", "src\xdpapi\xdpapi.vcxproj", "{0CCECB60-0538-4252-8C8E-23A92199CBE0}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "util", "test\common\lib\util\util.vcxproj", "{BDD99A80-0936-47B0-918D-04CF3B472FB0}"
```

Comment:
Anything under test should not be built in onebranch, because the tools required to build some subsets of our test code are not available there. It's simpler to just build the test code as part of the test workflow.